scripts/docs.sh

```diff
@@ -22,6 +22,21 @@ function generate_markdown {
     done;
 }
 
-go get github.com/devnev/godoc2ghmd
-generate_markdown
-echo "returning $?"
+function generate {
+    go get github.com/devnev/godoc2ghmd
+    generate_markdown
+    echo "returning $?"
+}
+
+function check {
+    generate
+    count=$(git diff --numstat | wc -l | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//')
+    echo $count
+    if [ "$count" = "0" ]; then
+        return 0
+    else
+        return 1
```

Comment:
Can you make this `echo` useful information such as:
"Your markdown docs seem to be out of sync with the package docs. Please run `make` and consult `CONTRIBUTING.MD`"